docs/Project.toml

```
@@ -3,6 +3,6 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 PythonOT = "3c485715-4278-42b2-9b5f-8f00e43c12ef"
 
 [compat]
-Documenter = "0.27"
+Documenter = "0.27, 1"
```

There were quite many breaking changes in Documenter 1, so I suggest dropping compatibility with Documenter < 1 (it doesn't matter for users anyway since it's only used in the environment for building the docs):
```suggestion
Documenter = "1"
```